### PR TITLE
Dockerfile: upgrade to ubuntu 14.04.1

### PR DESCRIPTION
**Pull request by bryanlatten**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -43,7 +34,19 @@ COPY ./apache2/conf-available/security.conf /etc/apache2/conf-available/security
 RUN apt-get autoclean && apt-get autoremove
 
 ADD ./container/run.sh /run.sh
+
 RUN chmod 755 /run.sh
 
+##################################################################
+
+# Move downstream application to final resting place
+ONBUILD COPY ./ /app/
```
Comment (bryanlatten) [inline Dockerfile:43]: for clarity, moved to bottom

File: Dockerfile [hunk 2]
```diff
@@ -43,7 +34,19 @@ COPY ./apache2/conf-available/security.conf /etc/apache2/conf-available/security
...
+
+# Remove existing docroot for Apache
+ONBUILD RUN rm -rf /var/www/html
+
+# Symlink existing docroot with downstream application docroot
+ONBUILD RUN ln -s /app/public /var/www/html
```
Comment (jimdelois) [inline Dockerfile]: Any reason not to combine these last two `RUN`s into a single one?

Comment (bryanlatten) [inline Dockerfile]: good point -- done

**Comment by jimdelois** [comment]
:+1:

**Pull request merged**